tests/command_line/test_import.py

```diff
@@ -134,6 +134,31 @@ def test_can_import_multiple_sequences(dials_data, tmp_path):
         assert experiment.identifier != ""
 
 
+def test_invert_axis_with_two_sequences_sharing_a_goniometer(dials_data, tmp_path):
+    # Test for regression of https://github.com/dials/dials/issues/2467
+    image_files = sorted(
+        dials_data("centroid_test_data", pathlib=True).glob("centroid*.cbf")
+    )
+    del image_files[4]  # Delete filename to force two sequences
+
+    result = subprocess.run(
+        [
+            shutil.which("dials.import"),
+            "output.experiments=experiments_multiple_sequences.expt",
+            "invert_rotation_axis=True",
+        ]
+        + image_files,
+        cwd=tmp_path,
+        capture_output=True,
+    )
+    assert not result.returncode and not result.stderr
+    assert (tmp_path / "experiments_multiple_sequences.expt").is_file()
+
+    experiments = load.experiment_list(tmp_path / "experiments_multiple_sequences.expt")
+    assert len(experiments.goniometers()) == 1
+    assert experiments.goniometers()[0].get_rotation_axis() == (-1.0, 0.0, 0.0)
```

Comment:
This test looks good / sane - what if however we also (manually) modify `experiments_multiple_sequences.expt` to have two turning forward, two turning backward and make sure they are all correctly inverted?

Reply:
I'm not sure what to do about that. The file `experiments_multiple_sequences.expt` only exists _after_ import, so it can't be changed to provide different input to `dials.import`. It's possible a rotation axis can be manually specified _and_ `invert_rotation_axis=True` be set. I'm not sure what currently happens in that case or indeed what _should_ happen.

Reply:
Ah should be able to do this with a direct test of `ManualGeometryUpdater` rather than testing `dials.import` as a command-line program. :+1: